hexl/CMakeLists.txt

```diff
@@ -97,7 +100,13 @@ else ()
     # For proper export of IntelHEXLConfig.cmake / IntelHEXLTargts.cmake,
     # we avoid explicitly linking dependencies via target_link_libraries, since
     # this would add dependencies to the exported hexl target.
-    add_dependencies(hexl cpu_features)
+
+    if (NOT CpuFeatures_FOUND)
```

Comment:
Nit: slightly more clear, IMO, would be to avoid the NOT, i.e.
```
if (CpuFeatures_FOUND)
  target_link_libraries(hexl PRIVATE cpu_features)
else()
  add_dependencies(hexl cpu_features)
endif()
```